plugins/search-backend/package.json

```diff
@@ -67,6 +67,7 @@
     "@backstage/backend-test-utils": "workspace:^",
     "@backstage/cli": "workspace:^",
     "@types/supertest": "^2.0.8",
+    "express-http-proxy": "^1.6.3",
```

Comment:
I did not see any usages of this package. is it needed here but not in catalog?

Reply:
an artifact from previous testing 😅 , will remove.